app/projectsmodel.cpp

```diff
@@ -544,17 +543,6 @@ void ProjectsModel::onProjectAttachedToMergin( const QString & )
   listProjectsByName();
 }
 
-void ProjectsModel::onAuthChanged()
-{
-  if ( !mBackend->userAuth() || !mBackend->userAuth()->hasAuthData() ) // user logged out, clear created and shared lists
-  {
-    if ( mModelType == CreatedProjectsModel || mModelType == SharedProjectsModel )
-    {
-      clearProjects();
```

Comment:
Don't we need to do this for the `WorkspaceProjectsModel`?

Reply:
I do not know.... if yes, it is bug in existing code?